fix(release-notes): adapt release-note related Gaja-X (#149)
catenax-eV/catenax-ev.github.io: src/pages/release-notes.md

### Further Scope Clarification

To uphold the integrity and reliability of the onboarding process, there will be a temporary change of the standards. This change is in-line with a feature supported in the current reference implementation.

Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [OSP](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles:

>   - The provided SD documents MUST be GAIA-X compliant, i.e. MUST provide a compliance credential issued from GAIA-X AISBL. – to be temporarily deactivated

Reviewer: ```suggestion
```

### Further Scope Clarification

To uphold the integrity and reliability of the onboarding process, there will be a temporary change of the standards. This change is in-line with a feature supported in the current reference implementation.

> Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [Onboarding service providers](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles within the following CACs:

Reviewer: ```suggestion
Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [OSP](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles:
```

> - [CX-0001](/docs/standards/CX-0001-EDCDiscoveryAPI#15-proof-of-conformity)

Reviewer: ```suggestion
```
